### DISCUSSION TITLE
Can we do marginal plot and partial plot by function “plot.variable”？

### DISCUSSION THREAD

**Discussion by A-Pai** [kogalur/randomForestSRC#175] [Q&A]
Can we do marginal plot and partial plot by function plot.variable”？
What is the difference between them？
![image](https://user-images.githubusercontent.com/5112397/149331363-0bb7e009-8923-4226-82b3-8fc71c19d77f.png)

**Comment by ishwaran** [comment]
Yes, the `plot.variable` (i.e. `plot.variable.rfsrc`) function plots either marginal (unadjusted) or partial (adjusted) plots.  Partial plots adjust for other variables.  Marginal plot is just a plot without adjustment.  See references in help file for more information.

**Reply by A-Pai** [reply]
Is there special literature about  partial plots？Is it “Partial Dependence Plots”？
Is this right？https://www.kaggle.com/dansbecker/partial-plots

**Comment by ishwaran** [comment]
Yes, as I mentioned, references are provided in the help file.  

Friedman J.H. (2001). Greedy function approximation: a gradient boosting machine, Ann. of Statist., 5:1189-1232.

**Reply by A-Pai** [reply]
what is meaning of “unadjusted“？---“Users can select between marginal (unadjusted, but fast) and partial plots (adjusted, but slower).”

**Comment by ishwaran** [comment]
See the new partial plot vignette:

https://luminwin.github.io/randomForestSRC/articles/partial.html
